### DISCUSSION TITLE
HTTPS not detected automatically

### DISCUSSION THREAD

**Discussion by markseuffert** [datenstrom/community#133] [Report a bug] [answered]
The webinterface should detect if a website uses http or https. Apparently this doesn't work on a https-only website and the user had to configure `WebinterfaceServerScheme: https`. It could be a bug in the webinterface initialisation or function `getServerScheme()`.

**Comment by wunderfeyd** [comment, marked as answer]
Should be fixed with commit e93d1e40cfe2323261feb6e300a7d5cded4d8f3c.
